steam_auto_cracker_gui.py

```diff
@@ -71,6 +72,7 @@ def DoRequest(self):
     def select_folder():
         global folder_path
         folder_path = filedialog.askdirectory()
+        folder_name = re.sub(r".*(\\|/)?([^\\]+)(\\|/)", "", folder_path)

```

Comment:
Using `os.path.basename` would probably be better

Reply:
True, regex was more intuitive for me tho

Reply:
Nice regex though haha, I hope it was found only or ChatGPT x)
They're always so hard to make

Reply:
Nah, i made it together with [https://regex101.com](https://regex101.com) 

They're alright when u get used to it